lib/oli_web/templates/project/publish.html.eex

```diff
@@ -10,11 +10,11 @@
           </div>
         <% {false, _} -> %>
           <div class="my-3">
-            Last published <strong><%= @latest_published_publication.published |> date(@conn) %></strong>.
+            Last published <strong><%= Utils.render_date(@latest_published_publication, :published, @context) %></strong>.
             There are <strong>no changes</strong> since the latest publication.
           </div>
         <% {true, changes} -> %>
-          <div class="my-3">Last published <strong><%= @latest_published_publication.published |> date(@conn) %></strong>.
+          <div class="my-3">Last published <strong><%= Utils.render_date(@latest_published_publication, :published, @context) %></strong>.
```

Comment:
Something that came to mind is that we could also accept a `conn` in the `render_date` function, initialize the context there, and avoid the init context call in phoenix controllers since we'll always have the connection in the template. Wdyt? 

I'm pretty good with what we have, not necessary to change but wanted to share the thought 😄

Reply:
I think it's a good idea 👍
I'll add it to the followup ticket and we can do it together with the other improvements if that works for you